DFC.ServiceTaxonomy.JobProfiles.Module/ServiceBusHandling/Converters/Helper.cs

```diff
@@ -108,5 +108,15 @@ contentItem.ContentType switch
                ContentTypes.ApprenticeshipStandard => contentItem.Content.ApprenticeshipStandard.LARScode.Text,
                _ => string.Empty,
            };
+
+        public static string GetHtmlWithoutInlineCss(dynamic html)
```

Comment:
The method name says without InlineCss but it's removing all html tags? Could perhaps use a better name? and is this code ported from DFC.Digital?